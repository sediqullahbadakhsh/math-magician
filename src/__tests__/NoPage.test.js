import renderer from 'react-test-renderer';
import NoPage from '../components/NoPage';

it('render correctly', () => {
  const tree = renderer.create(<NoPage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
