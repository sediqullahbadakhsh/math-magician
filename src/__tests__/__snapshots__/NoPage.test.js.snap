// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="nopage"
>
  <h2>
    404 page not found
  </h2>
  <h3>
    Please check your link :)
  </h3>
</div>
`;
